File: Data Structures & Algorithms/score-of-a-string/submission-1.js
class Solution {
    /**
     * @param {string} s
     * @return {number}
     */
    scoreOfString(s) {
        // split the each character in like 2
        // and then we like get the difference and add to the final
        // Stop at s.length - 1 so s[i + 1] never looks out of bounds!
        let res = 0
        for (let i = 0; i < s.length - 1; i++) {
            let sum = Math.abs(s.charCodeAt(i) - s.charCodeAt(i+1))
            res+=sum
        }
        return res
    }
}

File: Data Structures & Algorithms/baseball-game/submission-1.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        res = []

        for num in operations:
            if num == "C":
                res.pop()

            elif num == "D":
                res.append(res[-1] * 2)

            elif num == "+":
                res.append(res[-1] + res[-2])
                
            else:
                res.append(int(num))
        return sum(res)